app/controllers/fitter_happier/fitter_happier_controller.rb

```diff
@@ -0,0 +1,30 @@
+module FitterHappier
+  class FitterHappierController < ActionController::Base
+    layout nil
+
+    def index
+      render(:plain => "FitterHappier Site Check Passed\n")
+    end
+
+    def site_check
+      time = Time.now.to_formatted_s(:rfc822)
+      render(:plain => "FitterHappier Site Check Passed @ #{time}\n")
+    end
+
+    def site_and_database_check
+      version = DatabaseCheck.schema_version
+      time = Time.now.to_formatted_s(:rfc822)
+      render(:plain => "FitterHappier Site and Database Check Passed @ #{time}\nSchema Version: #{version}\n")
+    end
+
+    private
+
+    def process_with_silence(*args)
```

Comment:
We tried to remove this and line 28 because we didn't know exactly what they were doing. It also doesn't look like `#process_with_silence` was being called by anyone. Our IDE flagged `#alias_method_chain` as a deprecated method, so we did spend some time investigating, but we were unable to make any conclusions.